imops/radon.py

```diff
@@ -12,6 +12,12 @@
 from .utils import normalize_num_threads
 
 
+try:
+    from numpy.lib.array_utils import normalize_axis_tuple
+except ModuleNotFoundError:
+    from numpy.core.numeric import normalize_axis_tuple
```

Comment:
create a compat.py file and move all such failsafe imports there. then just import from it